src/Label.php

```diff
@@ -118,40 +164,40 @@ public static function post_action_label($post_id) {
 
     $data['attributes'] = $attributes;
 
-    $qr_code_size = $labelFormat[1] === 'landscape' ? self::QR_CODE_SIZE_LANDSCAPE : self::QR_CODE_SIZE_PORTRAIT;
+    $qr_code_size = $label_format[1] === 'landscape' ? self::QR_CODE_SIZE_LANDSCAPE : self::QR_CODE_SIZE_PORTRAIT;
     $qr_code = static::getProductQrCode($post_id, $qr_code_size);
 
     $data['qr_code'] = 'data:image/png;base64,' . base64_encode($qr_code);
 
-    return Pdf::render($data, $labelFormat[0], $labelFormat[1]);
+    return Pdf::render($data, $label_format[0], $label_format[1]);
   }
 
   /**
    * Adds price label printing controls to a given product.
    *
    * @param int $product_id
    *   The product unique identifier.
-   * @param string $labelFormatDefault
+   * @param string $label_format_default
    *   The default format for the price label.
    * @param bool $enabled
    *   If TRUE the controls are enabled.
    */
-  public static function addProductPriceLabelControls($product_id, $labelFormatDefault, $enabled = TRUE) {
-    Label::displayPriceLabelFormatsSelect(Label::PDF_LABEL_FORMATS, $labelFormatDefault, $enabled);
+  public static function addProductPriceLabelControls($product_id, $label_format_default, $enabled = TRUE) {
```

Comment:
Shouldn't an parameter named "default" have a default value? Also, why doesn't it have one here but further doen in the `displayPriceLabelFormatsSelect`?